core/unix/include/stat.h

```diff
@@ -0,0 +1,58 @@
+/****************************************************************************
+ ****************************************************************************
+ ***
+ ***   This header was generated from kernel headers to make
+ ***   information necessary for userspace to call into the Linux
+ ***   kernel available to DynamoRIO.  It contains only constants,
+ ***   structures, and macros generated from the original header, and
+ ***   thus, contains no copyrightable information.
+ ***
+ ****************************************************************************
+ ****************************************************************************/
+
+#ifndef _STAT_H
```

Comment:
I know the convention in DR (which maybe we should change) is just the file basename but this seems very short and like it might collide: maybe `s/STAT/CORE_UNIX_INCLUDE_STAT/`?